_product/ce/thor96/README.md

```diff
@@ -0,0 +1,113 @@
+product_specification: ce
+featured_product_order: 1
+is_featured: true
+product_images:
+  - thor96-sd-front.jpg
+  - thor96-sd-back.jpg
+# Sub Tab Menu Items
+tab_menu:
+    - tab_title: Thor96
+      tab_link: /product/thor96/
+      active: true
+    - tab_title: Getting Started
+      tab_link: /documentation/consumer/thor96/getting-started/
+      tab_position: 1
+    - tab_title: Documentation
+      tab_link: /documentation/consumer/thor96/
+      tab_position: 2
+      tab_position: 3
+    - tab_title: Support
+      tab_link: https://discuss.96boards.org/c/products/
+      tab_position: 4
+      tab_align_right: true
+product_getting_started: /documentation/consumer/thor96/getting-started/
+product_documentation_link: /documentation/consumer/thor96/
+product_support_link: "https://discuss.96boards.org/c/products/"
+
+# Buy Links
+product_buy_links:
+  -
+    link-title: Thor96
+    link-url: "https://www.arrow.com/en/products/i.imx8-thor96/arrow-development-tools"
+    from: Arrow Electronics
+    type: board
+    link-price: "$368.25"
+    link-price-currency: USD
+
+# Right Sidepanel links
+product_os:
+  - title: All Downloads
+    link: /documentation/consumer/thor96/downloads/
+
+product_more_info:
+  - title: Hardware User Manual
+    link: https://www.96boards.org/documentation/consumer/thor96/hardware-docs/files/thor96-hw-user-manual.pdf
+  - title: Schematics
+  - link: https://www.96boards.org/documentation/consumer/thor96/hardware-docs/files/thor96-schematics.pdf
+  - title: Arrow Campaign
+    link: https://www.arrow.com/en/campaigns/avengers
+
+# Bottom Link
+product_accessories:
+  - title: Power
+    link: /product/power/
+  - title: Adapter
+    link: /product/adapter/
+  - title: Debug
+    link: /product/debug/
+  - title: Misc
+    link: /product/misc/
+
+product_mezzanine:
+  - title: D3 Camera Mezzanine
+    link: /product/d3camera/
+  - title: Link Sprite Sensor Kit
+    link: /product/linkspritesensorkit/
+  - title: UART Serial
+    link: /product/uartserial/
+  - title: Sensors Mezzanine
+    link: /product/sensors-mezzanine
+  - title: Audio Mezzanine
+    link: /product/audio-mezzanine/
+
+product_kits:
+  - title: Coming Soon...
+    link: https://www.96boards.org/
+product: true
+archived: false
+---
+The Thor96 Board is a single-board computer powered by the NXP i.MX 8M SoC, incorporating a quad-core 64-bit Arm-A53, dedicated GPU and VPU, 4K support, Wi-Fi, Bluetooth and a wide range of I/O. The iMX 8M SoC is a feature-rich system containing a quad-core, 64-bit Arm A53 processor, Vulkan GPU with four shader cores and VPU capable of decoding 4K video at 60 fps. These features alone make the Thor96 board highly capable in a wide range of applications involving video and high processor requirements, including robotics, local AI systems, monitoring and drones.
+***
```

Comment:
what are these for?

Reply:
thats the writeup that is rendered on the actuall site
![Screenshot from 2020-01-13 19-07-46](https://user-images.githubusercontent.com/5900740/72260118-1e248600-3638-11ea-8864-040e80e3fec2.png)